src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilder.cs

```diff
@@ -1,38 +1,20 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 using Umbraco.Cms.Core.Composing;
 using Umbraco.Cms.Core.Events;
 using Umbraco.Cms.Core.Notifications;
-using Umbraco.Cms.Core.Webhooks.Events.Content;
-using Umbraco.Cms.Core.Webhooks.Events.DataType;
-using Umbraco.Cms.Core.Webhooks.Events.Dictionary;
-using Umbraco.Cms.Core.Webhooks.Events.Domain;
-using Umbraco.Cms.Core.Webhooks.Events.Language;
-using Umbraco.Cms.Core.Webhooks.Events.Media;
-using Umbraco.Cms.Core.Webhooks.Events.MediaType;
-using Umbraco.Cms.Core.Webhooks.Events.Member;
-using Umbraco.Cms.Core.Webhooks.Events.MemberType;
-using Umbraco.Cms.Core.Webhooks.Events.Package;
-using Umbraco.Cms.Core.Webhooks.Events.PublicAccess;
-using Umbraco.Cms.Core.Webhooks.Events.Relation;
-using Umbraco.Cms.Core.Webhooks.Events.RelationType;
-using Umbraco.Cms.Core.Webhooks.Events.Script;
-using Umbraco.Cms.Core.Webhooks.Events.Stylesheet;
-using Umbraco.Cms.Core.Webhooks.Events.Template;
-using Umbraco.Cms.Core.Webhooks.Events.User;
 using Umbraco.Extensions;
 
 namespace Umbraco.Cms.Core.Webhooks;
 
-public class WebhookEventCollectionBuilder : OrderedCollectionBuilderBase<WebhookEventCollectionBuilder,
-    WebhookEventCollection, IWebhookEvent>
+public class WebhookEventCollectionBuilder : OrderedCollectionBuilderBase<WebhookEventCollectionBuilder, WebhookEventCollection, IWebhookEvent>
```

Comment:
Why is this an ordered collection? The UI now has a search bar and orders it by `eventName`: https://github.com/umbraco/Umbraco-CMS/pull/15161#issuecomment-1829687564.

If we don't really do anything with the order, we could change the base class to `SetCollectionBuilderBase` and all `Append<T>()` calls to `Add<T>()` (I already have these changes working locally, so can push a commit if we agree on this).

Reply:
Yea I don't really a have a good reason for it to be OrderedCollection 🙈 The order of items doesn't really matter, as we're always looping through anyways 🤷